src/java/org/apache/cassandra/schema/SchemaKeyspace.java

```diff
@@ -552,7 +554,8 @@ private static void addTableParamsToRowBuilder(TableParams params, Row.SimpleBui
                .add("compaction", params.compaction.asMap())
                .add("compression", params.compression.asMap())
                .add("read_repair", params.readRepair.toString())
-               .add("extensions", params.extensions);
+               .add("extensions", params.extensions)
+               .add("disabled_automated_repair", params.disableAutomatedRepair);
```

Comment:
:+1: thanks for doing this.   I imagine it could be useful to be able to configure things more granularly.   I have some cases where we want to run full repair on a table, but never incremental (the main use case is analytics based where etl processes are loading a bunch of SSTables every few hours).   Will come back to this as I review and experiment more with the PR, if there is a bunch of things that we would want configurable at table level maybe we could consider adding something like a RepairParams, otherwise maybe this is fine as is.

Reply:
This may be overkill and some of the options may not be applicable, but wonder if we could just have a single field that functions similarly to `AutoRepairConfig.repair_type_overrides`:

So one could do something like:

```
ALTER TABLE x.y WITH automated_repair={'incremental': {'enabled': false}};
```

Most of the `repair_type_overrides` aren't really applicable at a table level, so maybe we'd just have a separate class like `AutoRepairParams` that functioned similarly to `CompactionParams`

Reply:
> 👍 thanks for doing this. I imagine it could be useful to be able to configure things more granularly. I have some cases where we want to run full repair on a table, but never incremental (the main use case is analytics based where etl processes are loading a bunch of SSTables every few hours). Will come back to this as I review and experiment more with the PR, if there is a bunch of things that we would want configurable at table level maybe we could consider adding something like a RepairParams, otherwise maybe this is fine as is.

Sure, makes sense. We can finalize all the configuration granularity that we want at a table level and then change it accordingly.

Reply:
I incorporated this comment. Now, we have more granular configuration support as follows:
```
    AND automated_repair_full = {'enabled': 'true'}
    AND automated_repair_incremental = {'enabled': 'true'};

```
Please note: I had to use two separate configurations instead of merging due to the limitation of the CQL grammar, which does not allow nested configuration for DDL statements.
```
    AND automated_repair = {'full': {'enabled': 'true'}, 'incremental': {'enabled': 'true'}}

```